crates/test-framework/src/snapshot/snapshots/results/test_framework__spicetest__datasets__worker__file[parquet]-sqlite[memory]_tpch_q6.snap

```diff
@@ -2,8 +2,8 @@
 source: crates/test-framework/src/spicetest/datasets/worker.rs
 description: "Query: tpch_q6"
 ---
-+---------------+
```

Comment:
Compared this change with other engines result and `123141078.2283` is correct new value